mysql/const.go

```diff
@@ -251,11 +251,11 @@ const (
 	// ConfigPriv is the privilege to enable the use SET CONFIG statements.
 	ConfigPriv
 
-	// AllPriv is the privilege for all actions.
-	AllPriv
-
 	// CreateTablespacePriv is the privilege to create tablespace.
 	CreateTablespacePriv
+
+	// AllPriv is the privilege for all actions.
```

Comment:
perhaps add a warning comment like #952 to ensure `AllPriv` is always the last value.

Reply:
OK